### DISCUSSION TITLE
WSL2 Remote: No suggestions/Unexpected end of JSON input

### DISCUSSION THREAD

**Discussion by coltonoscopy** [community/community#36124] [Copilot Conversations]
Hey there,

When using Copilot through the official Microsoft WSL Remote extension on my Windows machine, I'm getting an infinite spinner on the Copilot icon when fetching autocomplete suggestions, and no suggestions ever come through. The extension works great on normal Windows, just seems to be failing when used through WSL. The best hint I can find as to what is causing the issue is the following output every time it tries to fetch a suggestion within the Output > Log (Window) tab:

```
[2022-10-14 04:03:24.833] [renderer1] [error] Unexpected end of JSON input: SyntaxError: Unexpected end of JSON input
    at JSON.parse (<anonymous>)
    at a.json (/home/coltonoscopy/.vscode-server/extensions/github.copilot-1.50.6974/dist/extension.js:2:32150)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (node:internal/process/task_queues:96:5)
```

And the only thing that comes out in the Output > Github Copilot tab itself is this, anytime text is entered:

```
[INFO] [default] [2022-10-13T12:55:57.526Z] [fetchCompletions] engine https://copilot-proxy.githubusercontent.com/v1/engines/copilot-codex
```

Help in getting this working again in WSL would be greatly appreciated.

EDIT: After the most recent Github Copilot (v1.53.7011) and VSCode (1.72.2) updates, the former error no longer shows up in the log, but suggestions still never show up and the spinner still loops infinitely.

**Comment by iamcjwilliams** [comment]
@coltonoscopy Our copilot team has been notified about this issue
